src/node_data.js

```diff
@@ -110,13 +110,8 @@ const initData = function(node, nodeName, key) {
  * @return {!NodeData} The NodeData for this Node.
  */
 const getData = function(node) {
-  let data = node['__incrementalDOMData'];
-
-  if (!data) {
-    data = importNode(node);
-  }
-
-  return data;
+  return node['__incrementalDOMData'] ||
```

Comment:
Not a fan of code like this. Should just write it normally. I'm not concerned if it saves a few bytes.